src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    orange,
+    yellow,
+    green,
+    blue,
+    purple,
```

Comment:
```suggestion
    RED,
    ORANGE,
    YELLOW,
    ...
```

элементы списка принято писать капсом